cuenca/resources/service_providers.py

```diff
@@ -0,0 +1,13 @@
+from typing import ClassVar
+
+from pydantic.dataclasses import dataclass
+
+from .base import Queryable, Retrievable
+
+
+@dataclass
+class ServiceProvider(Retrievable, Queryable):
```

Comment:
what about categories?

Reply:
@kerycdiaz and I identified that categories might not be necessary for `cuenca-python`. He can explain the idea better tho

Reply:
Yes, although if you say they may be needed, we add them.

Obviously in Oaxaca if they should be, but in my opinion it may be needed in `cuenca-dart`